core/pom.xml

```diff
@@ -42,14 +42,17 @@
       <groupId>io.github.openfeign</groupId>
       <artifactId>feign-core</artifactId>
     </dependency>
+
```

Comment:
could we revert all changes from this file to keep this PR cleaner ?

Reply:
ok

Reply:
done